frontend/src/utils/profanity.ts

```diff
@@ -0,0 +1,5 @@
+const Filter = require('bad-words');
```

Comment:
I think here you can put `import Filter from 'bad-words';` instead to make it consistent with the other imports.